src/overlays/generic_sdfg_overlay.js

```diff
@@ -27,4 +27,5 @@ GenericSdfgOverlay.OVERLAY_TYPE = {
     MEMORY_VOLUME: 'OVERLAY_TYPE_MEMORY_VOLUME',
     STATIC_FLOPS: 'OVERLAY_TYPE_STATIC_FLOPS',
     RUNTIME_US: 'OVERLAY_TYPE_RUNTIME_US',
+    CONSTRUCTION: 'OVERLAY_TYPE_UNDER_CONSTRUCTION',
```

Comment:
you don't need any of this in webclient now that we can register general objects as overlays.

Reply:
The overlay isn't actually required anymore at all and was forgotten in here - it has now been removed.